### ISSUE TITLE
Driver does not work with docker-machine 0.5.5

### ISSUE THREAD

**Issue by legal90** [Parallels/docker-machine-parallels#32]
Reported by @leedm777
Moved from https://github.com/Parallels/docker-machine/issues/10

```
$ brew switch docker-machine 0.5.4
Cleaning /usr/local/Cellar/docker-machine/0.5.4
Cleaning /usr/local/Cellar/docker-machine/0.5.5
4 links created for /usr/local/Cellar/docker-machine/0.5.4

$ docker-machine ls
NAME          ACTIVE   DRIVER       STATE     URL                       SWARM   DOCKER    ERRORS
default       *        parallels    Running   tcp://10.211.55.70:2376           v1.9.1    
dev           -        virtualbox   Stopped                                     Unknown   
dns-testing   -        parallels    Stopped                                     Unknown   

$ brew switch docker-machine 0.5.5
Cleaning /usr/local/Cellar/docker-machine/0.5.4
Cleaning /usr/local/Cellar/docker-machine/0.5.5
4 links created for /usr/local/Cellar/docker-machine/0.5.5

$ docker-machine ls
Error attempting call to get driver name: connection is shut down
Error attempting call to get driver name: connection is shut down
NAME          ACTIVE   DRIVER       STATE     URL   SWARM   DOCKER    ERRORS
default       -                     Error                   Unknown   unexpected EOF
dev           -        virtualbox   Stopped                 Unknown   
dns-testing   -                     Error                   Unknown   
Error closing a plugin driver: connection is shut down
Error closing a plugin driver: connection is shut down
```

**Comment by legal90** [comment]
It is caused by https://github.com/docker/machine/pull/2636
We should to rebuild the plugin with `libmachine` v0.5.5 and release a new version

**Comment by legal90** [comment]
Fixed in [v1.1.1](https://github.com/Parallels/docker-machine-parallels/releases/tag/v1.1.1)
